assets/bingo/lang/pl_pl.json

```diff
@@ -0,0 +1,411 @@
+{
+	"%1074992263$s%1$s": "%2$s",
+	"bingo.technical.detect_multiplayer": "DoNotTranslateThis",
+
+	"bingo.resourcepack_check.sign.line1": "",
+	"bingo.resourcepack_check.sign.line2": "Kliknij",
+	"bingo.resourcepack_check.sign.line3": "Mnie",
+	"bingo.resourcepack_check.sign.line4": "",
+	"bingo.resourcepack_check.go_to_lobby.part1": "Paczka zasobów aktywowana pomyślnie.",
+	"bingo.resourcepack_check.go_to_lobby.part2": "%s aby przejść do lobby.",
+	"bingo.resourcepack_check.go_to_lobby.click": "Kliknij tutaj",
+
+	"bingo.game.go_to_lobby": "%s aby wrócić do lobby.\n%s aby zmienić na tryb widza.",
```

Comment:
Microsoft Word suggests adding a comma after "Kliknij tu"/"Kliknij tutaj" (represented as %s here). Of course no idea if that's correct :D

Reply:
Yes, the punctuation rule for [comma before `aby`]